internal/pkg/clusterresources/clusterresources_test.go

```diff
@@ -291,6 +292,79 @@ func TestIDMS_ITMSGenerator(t *testing.T) {
 	}
 }
 
+// verifyNoStatusField checks that a YAML file does not contain a 'status' field
+func verifyNoStatusField(t *testing.T, filePath, fileType string) {
+	t.Helper()
+	if _, err := os.Stat(filePath); err != nil {
+		// File doesn't exist, nothing to check
+		return
+	}
+
+	content, err := os.ReadFile(filePath)
+	if err != nil {
+		t.Fatalf("failed to read %s file: %v", fileType, err)
+	}
+
+	var yamlData map[string]any
+	err = yaml.Unmarshal(content, &yamlData)
+	if err != nil {
+		t.Fatalf("failed to parse %s YAML: %v", fileType, err)
+	}
```

Comment:
You can use [parser.ParseYamlFile](https://github.com/openshift/oc-mirror/blob/main/internal/pkg/parser/parse.go#L39-L40)